### ISSUE TITLE
Trouble replicating 77.0% accuracy with the quantized version of EfficientNet-EdgeTPU-S

### ISSUE THREAD

**Issue by sebtfa** [tensorflow/tpu#933]
Hi,

I have been evaluating the quantized .tflite version of [EfficientNet-EdgeTPU-S](https://github.com/tensorflow/tpu/tree/b24729de804fdb751b06467d3dce0637fa652060/models/official/efficientnet/edgetpu) against the ImageNet validation set on my CPU. I have only managed to reach 51.58% accuracy and can't find why I am not seeing 77.0% accuracy.

I have found that [normalizing](https://github.com/tensorflow/tpu/blob/b24729de804fdb751b06467d3dce0637fa652060/models/official/efficientnet/main.py#L369) the images by subtracting and dividing the image RGB values by the mean and standard deviation RGB values has only decreased the accuracy for the quantized version of the model. My general preprocessing approach has been cropping the image from the center and resizing the image to 224 by 224, which I got from [here.](https://github.com/tensorflow/tpu/blob/b24729de804fdb751b06467d3dce0637fa652060/models/official/efficientnet/preprocessing.py#L204)

I believe that the reason for such low accuracy may be due to the preprocessing method that I am using but I am interested in receiving feedback on how to increase the accuracy when evaluating this model. Any advice?

Thank you
